cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -95,20 +95,35 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
     generateConsole = false
 
     prepareAndTestQuery(
-      title = "Example of listing constraints",
+      title = "Listing all constraints",
       text =
         """
-          |To list all constraints with the brief output columns, the `SHOW CONSTRAINTS` command can be used.
-          |If all columns are wanted, use `SHOW CONSTRAINTS VERBOSE`.
-          |Filtering the output on constraint type is available for all types, the filtering keywords are listed in the <<administration-constraints-syntax, syntax table>>.
-          |As an example, to show only unique constraints, use `SHOW UNIQUE CONSTRAINTS`.""".stripMargin,
+          |To list all constraints with the default output columns, the `SHOW CONSTRAINTS` command can be used.
+          |If all columns are wanted, use `SHOW CONSTRAINTS YIELD *`.""".stripMargin,
       queryText = "SHOW CONSTRAINTS",
       prepare = _ => executePreparationQueries(List("CREATE CONSTRAINT ON (book:Book) ASSERT book.isbn IS UNIQUE")),
       optionalResultExplanation =
         """One of the output columns from `SHOW CONSTRAINTS` is the name of the constraint.
           |This can be used to drop the constraint with the <<administration-constraints-drop-constraint, `DROP CONSTRAINT` command>>.""".stripMargin,
       assertions = p => assert(p.size == 1)
     )
+
+    prepareAndTestQuery(
+      title = "Listing constraints with filtering",
+      text =
+        """
+          |One way of filtering the output from `SHOW CONSTRAINTS` on constraint type is the type keywords,
+          |these filtering keywords are listed in the <<administration-constraints-syntax, syntax table>>.
```

Comment:
```suggestion
          |One way of filtering the output from `SHOW CONSTRAINTS` by constraint type is the type keywords,
          |listed in the <<administration-constraints-syntax, syntax table>>.
```
Suggestion to make this a bit shorter.
Also, this link goes to the top of the syntax table, which is `CREATE CONSTRAINT` - I don't know if you can link to the correct table row?

Reply:
I don't think we have links to the table rows, just the table as a whole

I had sch problem trying to formulate this sentence, glad you found a better version!

Reply:
I think we do the same thing for indexes